app/views/sciruby_github/repos.erb

```diff
@@ -2,18 +2,38 @@
 <div class="span12 pagination-centered" style="text-align: center;">
     <h3> SciRuby github data analysis (Using daru-view/googlecharts)</h3>
     <b>Data is taken from url : https://api.github.com/orgs/Sciruby/repos </b>
-    <%= render 'export' %>
+    <%=
+      daru_download formats: %w[csv xls json], separator: '|'
+    %>
     <p style="left: 50%">
       <b>SciRuby Datatable : </b>
-      <%=raw @df_repo_table.div %>
+      <%=
+        daru_table @df_repos,
+                   adapter: :googlecharts,
+                   pageSize: 10,
+                   height: 300,
+                   width: 400
```

Comment:
Thanks for the ideas. 

We can have `daru_chart` , `daru_table` helpers for rails (refer [this](https://github.com/winston/google_visualr/blob/master/lib/google_visualr/app/helpers/view_helper.rb#L13) how `render_chart` can be used in rails app. Similarly we can have  `daru_table`, `daru_chart` ) , Sinatra (refer [sinatra helpers](https://www.sitepoint.com/using-sinatra-helpers-to-clean-up-your-code/)  and [sinatra_extension](http://www.sinatrarb.com/extensions.html) that seems helpful for it ),  not sure about how similar things can be done in Nanoc web applications.